gridfinity-rebuilt-bins.scad

```diff
@@ -25,15 +25,15 @@ https://github.com/kennetek/gridfinity-rebuilt-openscad
 
 /* [Setup Parameters] */
 $fa = 8;
-$fs = 0.25;
+$fs = 0.25; // .01
 
 /* [General Settings] */
 // number of bases along x-axis
-gridx = 3;
+gridx = 3; //.5
```

Comment:
I think I see what you're going for with these comments, but without any additional text it looks like you had temporary values you forgot to remove. I could see these comments getting deleted by someone inadvertently.

Is there any harm in using `3.0` instead of `3` here? I think that'd convey the intent that these can be floating point values without needing a comment.

Reply:
In my experiments `3.0` didn't change anything compared to `3`. Only the special comment did.

But I see your point about someone mistaking these comments as forgotten debugging values. I added a note in the comment block in the top of the file, see new commit. Let me know if it's visible and clear enough.